src/js/otus/components/Detail/Schema/Schema.tsx

```diff
@@ -30,15 +31,46 @@ export default function Schema({ match }: SchemaProps) {
         ReferenceRight.modify_otu,
     );
     const history = useHistory();
-
     const { data, isLoading } = useFetchOTU(otuId);
+    const mutation = useUpdateOTU();
+    const queryClient = useQueryClient();
 
     if (isLoading || isLoadingPermission) {
         return <LoadingPlaceholder />;
     }
 
     const { abbreviation, name, schema } = data;
 
+    function handleMoveUp(index: number) {
+        const updatedSchema = [...data.schema];
+        const temp = updatedSchema[index];
+        updatedSchema[index] = updatedSchema[index - 1];
+        updatedSchema[index - 1] = temp;
```

Comment:
nitpick: I like the shorter es6 syntax,  both are fine though tbh
```suggestion
        const updatedSchema = data.schema.slice();
        [updatedSchema[index], updatedSchema[index-1]] = [updatedSchema[index-1], updatedSchema[index]] 
```